src/main/scala/org/holmesprocessing/totem/services/asnmeta/asnmeta.py

```diff
@@ -99,8 +105,11 @@ def __init__(self):
 
 def main():
     server = tornado.httpserver.HTTPServer(ASNApp())
-    server.listen(Config.settings.port)
-    tornado.ioloop.IOLoop.current().start()
+    server.listen(Config["settings"]["port"])
+    try:
+        tornado.ioloop.IOLoop.instance().start()
```

Comment:
Documentation of tornado reads 
>"You should use `IOLoop.current` as the default when constructing an asynchronous object like HTTP Server. and use `IOLoop.instance` when you mean to communicate to the main thread from a different one."

 Since we don't have any threads in here, it's better we prefer `current()`